lib/job.js

```diff
@@ -14,7 +14,13 @@ var Job = module.exports = function Job(args) {
   delete args.agenda;
 
   // Process args
-  args.priority = parsePriority(args.priority) || 0;
+
+  // allow priority overrides (e.g. `job.now('foo', { priority: 'low' })`)
```

Comment:
Unfortunately I don't agree with this change as there's already a [method to change priority](https://github.com/agenda/agenda#prioritypriority) in the job API. Introducing the concept of reserved words and having Agenda's behaviour leak into the data object will just cause problems IMO. For example, breaking someone's job because their domain data also has a priority key.